beacon-chain/casper/sharding_test.go

```diff
@@ -116,11 +120,136 @@ func TestSampleAttestersAndProposers(t *testing.T) {
 		t.Errorf("Get attesters failed, expected: %v got: %v", validatorList[:len(attesters)], attesters)
 	}
 
-	indices, err = ValidatorsByHeightShard(common.Hash{'A'}, validators, 1, 0)
+	indices, err := ShuffleValidatorsToCommittees(common.Hash{'A'}, validators, 1, 0)
 	if err != nil {
 		t.Errorf("validatorsByHeightShard failed with %v:", err)
 	}
-	if len(indices) != 8192 {
-		t.Errorf("incorret length for validator indices. Want: 8192. Got: %v", len(indices))
+	if len(indices) != params.CycleLength {
+		t.Errorf("incorret length for validator indices. Want: %d. Got: %d", params.CycleLength, len(indices))
+	}
+}
+
+func TestGetCommitteeParamsSmallValidatorSet(t *testing.T) {
```

Comment:
Can you elaborate on what you mean? These different parameters test different code paths.

Reply:
Oh, I think you're saying use a helper function to dedupe code. If so, I generally try to avoid helper functions in tests. They make the test harder to read and overall more difficult to maintain. In this case, Especially here, where I'm only testing 3 different sets of parameters.

Reply:
No, i'm saying define to use table driven test. Sorry I wrote "test driven test" lol.

Reply:
Ohhh. Hmm, I'd consider it if there were more than 3 test cases. Seems optional for this one.